go/cmd/vtorc/main.go

```diff
@@ -106,6 +106,7 @@ func main() {
 	grpccommon.RegisterFlags(fs)
 	vtlog.RegisterFlags(fs)
 	logutil.RegisterFlags(fs)
+	servenv.RegisterMySQLServerFlags(fs)
```

Comment:
I don't think VTOrc requires this flag

Reply:
```
go list -f '{{ .ImportPath }}| {{ join .Deps " " }} ' ./go/vt/orchestrator/... | awk -F"|" '$2 ~ " vitess.io/vitess/go/vt/sqlparser " {print $1}'
vitess.io/vitess/go/vt/orchestrator/app
vitess.io/vitess/go/vt/orchestrator/discovery
vitess.io/vitess/go/vt/orchestrator/http
vitess.io/vitess/go/vt/orchestrator/inst
vitess.io/vitess/go/vt/orchestrator/logic
```

just taking one:

`go/vt/orchestrator/logic` imports `go/vt/vttablet/tmclient` and instantiates a `TabletManagerClient`, which imports `go/vt/mysqlctl/tmutils`, which imports `go/vt/schema`, which, finally, imports and uses `sqlparser.Parse*` functionality.